src/backend/src/utils/slack.utils.ts

```diff
@@ -37,3 +37,22 @@ export const sendSlackUpcomingDeadlineNotification = async (workPackage: WorkPac
 
   await sendMessage(LEAD_CHANNEL_SLACK_ID, fullMsg);
 };
+
+export const sendSlackRequestedReviewNotification = async (slackId: string, changeRequest: ChangeRequest): Promise<void> => {
+  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
+
+  const { requestedReviewers } = changeRequest;
+  const slackIds = requestedReviewers.map(async (reviewer) => await getUserSlackId(reviewer.userId));
```

Comment:
your already looping this, thats why the given slackId in the paramater is the slackId you want